### ISSUE TITLE
Does not run with ArduinoJson 6?

### ISSUE THREAD

**Issue by r0oland** [platformio/bintray-secure-ota#11]
When I try to compile, I get the following error:
```
lib\BintrayClient\src\BintrayClient.cpp:115:11: error: StaticJsonBuffer is a class from ArduinoJson 5. Please see arduinojson.org/upgrade to learn how to upgrade your program to ArduinoJson version 6
     StaticJsonBuffer<bufferSize> jsonBuffer;
```
Now I guess the error is my "wrong" ArduinoJson version, but wouldn't it be nice if your code ran with the latest ArduinoJson version?

**Comment by r0oland** [comment]
If anyone has the same issue as me and needs to force the use of the old Jason lib, you just have to change your platformio.ini to:
`lib_deps =   ArduinoJson@<6.9
`
This forces the use of the older json lib...

**Comment by r0oland** [comment]
Fixed with commit 148b7a447f639cd29da4fb4e497ffc62780d18ae
